### PR TITLE
Finish basic graphql schema setup

### PR DESCRIPTION
**Pull request by denvaar**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 language: elixir
 
 elixir:
-  - 1.6.5
+  - 1.7.3
```
Comment (alex-kovshovik) [inline .travis.yml:4]: 👍

File: config/show_n_tell.exs [hunk 2]
```diff
@@ -1,4 +1,4 @@
 use Mix.Config
 
 config :show_n_tell,
-  github_org: System.get_env("GITHUB_ORG") || "decisely"
+  github_org: "decisely"
```
Comment (alex-kovshovik) [inline config/show_n_tell.exs:4]: Why remove this configurability? We could let other orgs use this app later...

File: lib/show_n_tell_web/router.ex [hunk 3]
```diff
@@ -5,14 +5,18 @@ defmodule ShowNTellWeb.Router do
...
+
   scope "/api", ShowNTellWeb do
     pipe_through :api
 
     resources "/authenticate", AuthController, singleton: true, only: [:create, :show]
   end
 
   scope "/" do
-    pipe_through :api
+    pipe_through [:api, :graphql]
```
Comment (alex-kovshovik) [inline lib/show_n_tell_web/router.ex:19]: Nice.

**Review by alex-kovshovik — APPROVED** [review]
Looks great! Thanks for doing it Denver! I'd like to set this API server up - to let other team members build clients against it. We can do this!!!

File: config/show_n_tell.exs (hunk 2 above)
Comment (denvaar) [reply config/show_n_tell.exs:4]: I will add it back.

Comment (denvaar) [reply config/show_n_tell.exs:4]: Actually, I think the best way is to use the mix config system to do this rather than an environment variable.

**Pull request merged**